bot.go

```diff
@@ -877,36 +876,49 @@ func (b *Bot) FileByID(fileID string) (File, error) {
 // Download saves the file from Telegram servers locally.
 //
 // Maximum file size to download is 20 MB.
-func (b *Bot) Download(f *File, localFilename string) error {
-	g, err := b.FileByID(f.FileID)
+func (b *Bot) Download(file *File, localFileName string) error {
+	f, err := b.FileByID(file.FileID)
 	if err != nil {
 		return err
 	}
 
 	url := fmt.Sprintf("%s/file/bot%s/%s",
-		b.URL, b.Token, g.FilePath)
+		b.URL, b.Token, f.FilePath)
 
-	out, err := os.Create(localFilename)
+	out, err := get(url)
 	if err != nil {
 		return wrapSystem(err)
 	}
-	defer out.Close()
 
-	resp, err := http.Get(url)
+	err = ioutil.WriteFile(localFileName, out, 0644)
```

Comment:
Old code was doing io.Copy from resp.Body, and didn't read everything into memory.
I'd say it's a regression.